### PR TITLE
Rename callback functions of metadata editor buttons

### PR DESCRIPTION
**Pull request by solth**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java [hunk 1]
```diff
@@ -89,7 +89,7 @@
      * This method is invoked if the user clicks on the add multi doc struc
      * submit btn command button.
      */
-    public void addMultiDocStrucSubmitBtnClick() {
+    public void addMultiDocStrucSubmit() {
```
Comment (Beacze) [inline Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java]: Is submit needed in the name?

File: Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java [hunk 2]
```diff
@@ -103,7 +103,7 @@ public void addMultiDocStrucSubmitBtnClick() {
      * This method is invoked if the user clicks on the add single doc struc
      * submit btn command button.
      */
-    public void addSingleDocStrucSubmitBtnClick() {
+    public void addSingleDocStrucSubmit() {
```
Comment (Beacze) [inline Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java]: Is submit needed in the name?

File: Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java (hunk 1 above)
Comment (solth) [reply Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java]: You are right, it's probably not needed!

File: Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java (hunk 2 above)
Comment (solth) [reply Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java]: You are right, it's probably not needed!

**Pull request merged**